vertical-pod-autoscaler/e2e/v1/common.go

```diff
@@ -294,10 +301,10 @@ func WaitForPodsRestarted(f *framework.Framework, podList *apiv1.PodList) error
 }
 
 // WaitForPodsEvicted waits until some pods from the list are evicted.
-func WaitForPodsEvicted(f *framework.Framework, podList *apiv1.PodList) error {
+func WaitForPodsEvicted(f *framework.Framework, podList *apiv1.PodList, timeout time.Duration) error {
```

Comment:
What is the reason for this change?
It seems like all occurrences of calls to `WaitForPodsEvicted` just pass in `utils.PollTimeout`